### ISSUE TITLE
Undefined symbol when importing GPflow

### ISSUE THREAD

**Issue by danstowell** [GPflow/GPflow#268]
I'm having an issue similar to #173. *(I'm raising a separate issue since it appears #173 was solved by #175, and I'm on a different linux flavour than in #173.)* Installing GPflow appears to succeed, but then when I try to use it, there's an "undefined symbol" error:

```
Using /usr/local/lib/python2.7/dist-packages
Finished processing dependencies for GPflow==0.3.4

 % python -c "import GPflow"
I tensorflow/stream_executor/dso_loader.cc:128] successfully opened CUDA library libcublas.so locally
I tensorflow/stream_executor/dso_loader.cc:128] successfully opened CUDA library libcudnn.so locally
I tensorflow/stream_executor/dso_loader.cc:128] successfully opened CUDA library libcufft.so locally
I tensorflow/stream_executor/dso_loader.cc:128] successfully opened CUDA library libcuda.so.1 locally
I tensorflow/stream_executor/dso_loader.cc:128] successfully opened CUDA library libcurand.so locally
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "GPflow/__init__.py", line 18, in <module>
    from . import likelihoods, kernels, param, model, gpmc, sgpmc, priors, gpr, svgp, vgp, sgpr, tf_wraps, tf_hacks
  File "GPflow/likelihoods.py", line 17, in <module>
    from . import densities, transforms
  File "GPflow/transforms.py", line 18, in <module>
    from . import tf_wraps as tfw
  File "GPflow/tf_wraps.py", line 32, in <module>
    _custom_op_module = tf.load_op_library(os.path.join(os.path.dirname(__file__), 'tfops', 'matpackops.so'))
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/load_library.py", line 64, in load_op_library
    None, None, error_msg, error_code)
tensorflow.python.framework.errors_impl.NotFoundError: GPflow/tfops/matpackops.so: undefined symbol: _ZN10tensorflow8internal21CheckOpMessageBuilder9NewStringEv

```

My system info:

* ubuntu 16.04
* GCC 5.4
* tensorflow 0.11.0rc2 installed from a "git clone" and built from source
* python 2.7.12
* NOT using virtualenv

I've installed the latest git master of GPflow (3f9c248) and `sudo python setup.py develop`. (Regarding #175, sys.platform for me is "linux2".)

**Comment by danstowell** [comment]
P.S. I'm using tensorflow from source not from binary, because the tensorflow instructions say the binaries don't support my cuda/cudnn version

**Comment by jameshensman** [comment]
Hi @danstowell , did you see Tom's note in #227 ? Seems related.

**Comment by danstowell** [comment]
Aha. It's not clear which user is "Tom", but if I comment out the two lines identified by @c0g then this error goes away. Looks like this issue can be subsumed into #227.

**Comment by alexggmatthews** [comment]
Fixed in #440
